spacer/tasks.py

```diff
@@ -3,8 +3,7 @@
 """
 import time
 import traceback
-from logging import getLogger
-
```

Comment:
This `getLogger` import is still needed. (This was something I worked with recently, so maybe this deletion was a merge/rebase slip-up?)